testing/cli/test_cli_sct_resample.py

```diff
@@ -31,3 +31,34 @@ def test_sct_resample_output_has_expected_dimensions(path_in, type_arg, dim, exp
 
     for actual_val, expected_val in zip(actual_dim, expected_dim):
         assert actual_val == pytest.approx(expected_val, abs=abs_tol)
+
+
+def test_sct_resample_no_f_mm_vox_ref_arguments(tmp_path, capfd):
+    """
+    Test if an error is raised if none of '-f', '-mm', '-vox' or '-reg' arguments are passed.
+    """
+    path_in = str(tmp_path / 't2/t2.nii.gz')
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        sct_resample.main(argv=['-i', path_in])
+    assert pytest_wrapped_e.value.code == 2
+
+    # Capture stdout and stderr
+    out, err = capfd.readouterr()
+    # Assert that the error message is printed to stderr
+    assert "You need to specify one of those four arguments" in err
```

Comment:
This is a really useful pattern that I hadn't seen before! I'll have to keep it in mind when writing other tests that use `parser.error` or `printv`.